Source/DataModels/Windows/SubWindows/ComponentWindows/WindowComponentPlayer.cpp

```diff
@@ -0,0 +1,20 @@
+#include "WindowComponentPlayer.h"
+
+#include "Components/ComponentPlayer.h"
+
+#include "Application.h"
+
+
+WindowComponentPlayer::WindowComponentPlayer(ComponentPlayer* component) :
+	ComponentWindow("PLAYER", component)
+{
+}
+
+WindowComponentPlayer::~WindowComponentPlayer()
+{
+}
+
+void WindowComponentPlayer::DrawWindowContents()
+{
+	DrawEnableAndDeleteComponent();
```

Comment:
Idk why but, when drawing the `PlayerComponent` Subwindow in the inspector, the `ImGui::Separator()`that is supposed to appear at the end of each component, in this case appears at the beginning, please take a look as it appears strange in the inspector window with all the other components.

Reply:
Nice catch, now I can see this too. It appears at the end of the Header but not in the end of the Component Window content.

Reply:
I realize what it happens. It is because the `DrawEnableAndDeleteComponent()` method uses the `SameLine()` and then there isn't more content to show in this `WindowComponentPlayer`. So, the `ImGui::Separator()` is displayed at same line. For avoiding this, you can use the follow code suggestion:
```suggestion
	DrawEnableAndDeleteComponent();
	ImGui::Text("");
```